recipes/jbig/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "2.1" %}
+
+package:
+    name: jbig
+    version: {{ version }}
+
+source:
+    fn: jbigkit-{{ version }}.tar.gz
+    url: https://www.cl.cam.ac.uk/~mgk25/jbigkit/download/jbigkit-{{ version }}.tar.gz
+    md5: ebcf09bed9f14d7fa188d3bd57349522
+    patches:
+        # Remove -Wno-unused-result (for compatibility with old gcc), and add -fPIC (to allow shared libraries to use this build).
+        - Makefile.patch
```

Comment:
I just test exporting  `CFLAGS="$CFLAGS -fPIC"`, but that does does not work. We do need the patch. (Or someone smarter than me.)